Gifski/util.swift

```diff
@@ -64,6 +64,55 @@ class SSView: NSView {
 }
 
 
+extension NSWindow {
+	static let defaultContentSize = CGSize(width: 480, height: 300)
+
+	static var defaultContentRect: CGRect = {
+		let size = defaultContentSize
+
+		guard let screen = NSScreen.main else {
+			return CGRect(origin: .zero, size: size)
+		}
+
+		let extraOffset = screen.visibleFrame.height * 0.19
+
+		return CGRect(
+			x: screen.visibleFrame.midX - (size.width / 2),
```

Comment:
@LarsJK Can you see anything wrong with my calculations here? When I remove `window.center()` below the window is not in the exact same `x` position...

![4](https://user-images.githubusercontent.com/170270/37580134-ab3b94e6-2b74-11e8-9815-73187c52147c.gif)

Reply:
The calculations look correct to me.. Do you have the dock at the bottom or to either side?

Reply:
No, the Dock is hidden.

Reply:
Ok, I solved it. I was stupid. I calculated based on the default window size, not our window size